pages/common/tldr.md

```diff
@@ -14,3 +14,7 @@
 - Get help for a git subcommand:
 
 `tldr {{git-checkout}}`
+
+- Update the local tldr pages:
+
+`tldr --update`
```

Comment:
```suggestion
`tldr -u`
```
Use short version since long version is not only optional to support for clients.